tcp.rflx

```diff
@@ -0,0 +1,124 @@
+package TCP is
+
+    -- RFC 793
+
+    type Source_Port is mod 2**16;
+    type Destination_Port is mod 2**16;
+    type Sequence_Number is mod 2**32;
+    type Acknowledgment_Number is mod 2**32;
+    type Data_Offset is range 5 .. 2**4 - 1 with Size => 4;
+
+    -- Reserved for future use. Must be zero.
+    type Reserved is range 0 .. 0 with Size => 3;
+
+    type Window is mod 2**16;
+    type Checksum is mod 2**16;
+    type Urgent_Pointer is mod 2**16;
+
+    -- Timestamp and Window_Scale options are described in RFC 7323
+    -- Selective Acknowledgment Options (SACK) and SACK_Permitted are described in RFC 2018
+    -- No_Operation, End_Of_Options_List are described in RFC 793
+    -- Maximum_Segment_Size is described in RFC 793 and RFC 879
+
+    type Kind is
+       (End_Of_Options_List    => 16#00#,
```

Comment:
In the RFC it is called `End_Of_Option_List`.

Reply:
Renamed in dc9f40f3b2fde6d4029438a000055e9658205250.